doc/changes/changes_9.0.0.md

```diff
@@ -1,11 +1,12 @@
-# script-languages-release 9.0.0, released t.b.d.
+# script-languages-release 9.0.0, released 2024-11-26
 
-Code name: t.b.d.
+Code name: New script options parser
 
 ## Summary
 
-t.b.d.
-
+This release introduces the new Script Options parser used in the Java UDF's. The new parser provides additional escape syntax which allows a more flexible definition of option values.
```

Comment:
the new parser is optional is important.

Reply:
Added a note